DTLOR_DP.py

```diff
@@ -119,41 +126,34 @@ def DP(hostTree, parasiteTree, phi, locus_map, D, T, L, Origin, R):
         dynamic programming algorithm are explained in the tech report.
```

Comment:
I think we are not currently returning the number of MPRs? Can you modify the comment to reflect the current outputs?